cache/solidity-files-cache.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do you mind adding `cache/*` to the `.gitignore`?

Reply:
Will need to remove this folder from your git cache: `git rm --cached cache/solidity-files-cache.json`